cli/test/support/testData.ts

```diff
@@ -35,7 +35,7 @@ export const insertTestData = async () => {
       </project/cli-test>
           a ${hydra.Resource} , ${cc.CubeProject} ;
           ${cc.csvMapping} </project/cli-test/mapping> ;
-          ${cc.cube}       </cube/cli-test>
+          ${cc.cubeGraph}  </cube/cli-test>
```

Comment:
I propose a `cc:cubeGraph` property on project to explicitly prepare the location for generated triples